File: FITCC.UI/mobile_fitcc/lib/Models/takmicenje.dart
import 'package:json_annotation/json_annotation.dart';

part 'takmicenje.g.dart';

@JsonSerializable(explicitToJson: true)
class Takmicenje {
  int takmicenjeId;
  String naziv;
  String slogan;
  DateTime pocetak;
  DateTime kraj;
  int godina;
  int brojDana;
  String? slika;

  Takmicenje(this.takmicenjeId, this.naziv, this.slika, this.pocetak, this.kraj,
      this.godina, this.brojDana, this.slogan);

  factory Takmicenje.fromJson(Map<String, dynamic> json) =>
      _$TakmicenjeFromJson(json);
  Map<String, dynamic> toJson() => _$TakmicenjeToJson(this);
}

File: FITCCRS2App.Models/Models/Takmicenje.cs
﻿namespace FITCCRS2App.Models.Models
{
	public class Takmicenje
	{
        public int TakmicenjeId { get; set; }
        public string Naziv { get; set; } = string.Empty;
        public string Slogan { get; set; } = string.Empty;
        public DateTime? Pocetak { get; set; }
        public DateTime? Kraj { get; set; }
        public int? Godina { get; set; }
        public int? BrojDana { get; set; }
        public string Slika { get; set; } = string.Empty;

        //public virtual ICollection<Agenda> Agenda { get; set; }
        //public virtual ICollection<Tim> Tims { get; set; }
    }
}

